src/providers/pdal/CMakeLists.txt

```diff
@@ -53,11 +110,35 @@ include_directories(SYSTEM
   ${PDAL_INCLUDE_DIR}
 )
 
-add_library (pdalprovider MODULE ${PDAL_SRCS} ${PDAL_HDRS})
+add_executable(untwine ${UNTWINE_SRCS} ${UNTWINE_HDRS})
+set_target_properties(untwine PROPERTIES
+  RUNTIME_OUTPUT_DIRECTORY ${QGIS_OUTPUT_DIRECTORY}/${QGIS_LIBEXEC_SUBDIR}
+)
+
+target_link_libraries (untwine
+  ${PDAL_LIBRARIES}
```

Comment:
I had to add `pthread` on linux to get untwine compiled (travis does not complain because it does not build PDAL provider I guess?)